packages/@okta/vuepress-theme-prose/assets/css/base/_typography.scss

```diff
@@ -9,13 +9,102 @@
   font-style: normal;
 }
 
+@font-face {
+  font-family: 'ABC Whyte';
+  src: url('/fonts/ABC/Light/ABCWhyte-Light.eot');
+  src: url('/fonts/ABC/Light/ABCWhyte-Light.eot?#iefix') format('embedded-opentype'),
+      url('/fonts/ABC/Light/ABCWhyte-Light.woff2') format('woff2'),
+      url('/fonts/ABC/Light/ABCWhyte-Light.woff') format('woff'),
+      url('/fonts/ABC/Light/ABCWhyte-Light.ttf') format('truetype'),
+      url('/fonts/ABC/Light/ABCWhyte-Light.svg#ABCWhyte-Light') format('svg');
+  font-weight: 300;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'ABC Whyte';
+  src: url('/fonts/ABC/Regular/ABCWhyte-Regular.eot');
+  src: url('/fonts/ABC/Regular/ABCWhyte-Regular.eot?#iefix') format('embedded-opentype'),
+      url('/fonts/ABC/Regular/ABCWhyte-Regular.woff2') format('woff2'),
+      url('/fonts/ABC/Regular/ABCWhyte-Regular.woff') format('woff'),
+      url('/fonts/ABC/Regular/ABCWhyte-Regular.ttf') format('truetype'),
+      url('/fonts/ABC/Regular/ABCWhyte-Regular.svg#ABCWhyte-Regular') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'ABC Whyte';
+  src: url('/fonts/ABC/Medium/ABCWhyte-Medium.eot');
+  src: url('/fonts/ABC/Medium/ABCWhyte-Medium.eot?#iefix') format('embedded-opentype'),
+      url('/fonts/ABC/Medium/ABCWhyte-Medium.woff2') format('woff2'),
+      url('/fonts/ABC/Medium/ABCWhyte-Medium.woff') format('woff'),
+      url('/fonts/ABC/Medium/ABCWhyte-Medium.ttf') format('truetype'),
+      url('/fonts/ABC/Medium/ABCWhyte-Medium.svg#ABCWhyte-Medium') format('svg');
+  font-weight: 500;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'ABC Whyte';
+  src: url('/fonts/ABC/Bold/ABCWhyte-Bold.eot');
+  src: url('/fonts/ABC/Bold/ABCWhyte-Bold.eot?#iefix') format('embedded-opentype'),
+      url('/fonts/ABC/Bold/ABCWhyte-Bold.woff2') format('woff2'),
+      url('/fonts/ABC/Bold/ABCWhyte-Bold.woff') format('woff'),
+      url('/fonts/ABC/Bold/ABCWhyte-Bold.ttf') format('truetype'),
+      url('/fonts/ABC/Bold/ABCWhyte-Bold.svg#ABCWhyte-Bold') format('svg');
+  font-weight: bold;
+  font-style: normal;
+  font-display: swap;
+}
+
 a {
+  color: cv('link');
   text-decoration: none;
+  &:visited, &:active, &:focus {
+    color: cv('link');
+  }
 }
 
+
+h1, h2, h3, h4, h5, h6 {
+  color: cv('text', 'darker');
+  margin-top: 3.125rem;
+}
+
+h1 {
+  margin-top: 30px;
+  font-size: 48px;
```

Comment:
Why are we hardcoding font sizes? They should be imported from Odyssey: https://odyssey.okta.design/base/typography/